src/app/app.module.ts

```diff
@@ -2,10 +2,12 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
+import { FooterComponent } from './core/footer/footer.component';
```

Comment:
You can set 
`{
  "compilerOptions": {
    "baseUrl": "src"
}`
then you will able to
`import { FooterComponent } from '**app**/core/footer/footer.component';`
in any file in src folder